README.rst

```diff
@@ -144,3 +144,12 @@ Citation
 To cite YASA, please use the `eLife publication <https://elifesciences.org/articles/70092>`_:
 
 * Vallat, Raphael, and Matthew P. Walker. "An open-source, high-performance tool for automated sleep staging." Elife 10 (2021). doi: https://doi.org/10.7554/eLife.70092
+
+Related Projects
+~~~~~~~~~~~~~~~~~
+
+Looking for a web-based solution to utilize YASA? Check out **YASA Flaskified**: a Flask-based application designed to process and analyze EEG data via an intuitive web interface. 
```

Comment:
I would prefer some kind of bullet list here, e.g.:

```rst
- `YASA Flaskified <https://github.com/bartromb/YASAFlaskified>`_: a Flask-based application designed to process and analyze EEG data via an intuitive web interface. YASA Flaskified also supports straightforward deployment on your own physical or virtual server using a preconfigured deployment script. 
```

Reply:
Agreed, a bulleted list is preferred. And actually I think that the bullet could be placed under the "Development" subheading. A section titled "Related Projects" might be overkill for just this, unless we plan to add other sleep analysis software there as well. If left as-is, I would probably add some other packages to the list later.